regimes/mx/invoice_validator_test.go

```diff
@@ -0,0 +1,99 @@
+package mx_test
+
+import (
+	"testing"
+
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/l10n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func validInvoice(t *testing.T) *bill.Invoice {
```

Comment:
you can add a `t.Helper()` call at the start of the method to avoid the linter unused param issue.

Reply:
I thought that the linter was right that I wasn't really using it, so I just removed it. But the trick is good to know